docs/fsharp/tools/fsharp-interactive/index.md

```diff
@@ -85,7 +85,7 @@ F# scripting is natively supported in [Visual Studio](../../get-started/get-star
 ## Referencing packages in F# Interactive
 
 > [!NOTE]
-> Package management is an F# 5 feature and is currently available using the latest .NET 5 SDK.
+> Package management is available using .NET 5 SDK or later.
```

Comment:
I would actually get rid of this note entirely; it's a default on the latest installation, so we can just assume it's available.

Reply:
I felt leaving it since dotnet 3.1 is an LTS release that might linger around, but I can remove it

Reply:
Removing it would probably be for the best. .NET Core 2.1 is also still technically an LTS, and there are several older versions of VS that have older F# compilers that are LTS, but we don't account for that in docs. It would be far too complicated to maintain separate guidance across different release channels in documentation like this.